README.md

```diff
@@ -84,7 +84,15 @@ pipe = pipe.to("cuda")
 
 prompt = "a photo of an astronaut riding a horse on mars"
 with autocast("cuda"):
-    image = pipe(prompt).images[0]  
+    pipeline_output = pipe(prompt)
```

Comment:
I think we want to keep our main examples as easy to understand as possible, so ideally we shouldn't change anything here :-)

Reply:
OK, I reverted them!